pkg/scheduler/framework/interface.go

```diff
@@ -181,9 +181,9 @@ type Status struct {
 	code    Code
 	reasons []string
 	err     error
-	// plugin is an optional field that records the plugin name causes this status.
+	// plugins is an optional field that records the plugins name causes this status.
 	// It's set by the framework when code is Unschedulable, UnschedulableAndUnresolvable or Pending.
-	plugin string
+	plugins []string
```

Comment:
Do we still have to have this change?

Reply:
yes, one node may be filtered by different `PreFilter`